File: core/src/ru/myitschool/dcrawler/entities/skills/patterns/FloorSingleTargetPattern.java
package ru.myitschool.dcrawler.entities.skills.patterns;

import ru.myitschool.dcrawler.entities.skills.Skill;
import ru.myitschool.dcrawler.entities.skills.Target;

public class FloorSingleTargetPattern extends TargetPattern {
    public FloorSingleTargetPattern(Skill skill) {
        super(skill);
    }

    @Override
    public Target createTarget(Target target) {
        return target;
    }
}

File: core/src/com/wolg_vlad/dcrawler/entities/skills/patterns/FloorSingleTargetPattern.kt
package com.wolg_vlad.dcrawler.entities.skills.patterns

import com.wolg_vlad.dcrawler.entities.skills.Skill
import com.wolg_vlad.dcrawler.entities.skills.Target

class FloorSingleTargetPattern(skill: Skill) : ValidatedTargetPattern(skill) {
    override fun createTarget(target: Target): Target {
        return target
    }
}